py3status/modules/bluetooth.py

```diff
@@ -106,16 +116,18 @@ def bluetooth(self):
         if devices:
             data = []
             for mac, name in devices:
-                data.append(self.py3.safe_format(self.format_device, {'name': name, 'mac': mac}))
-            full_text = self.py3.composite_join(self.device_separator, data)
+                data.append(self.py3.safe_format(
+                    self.format_device, {'name': name, 'mac': mac}))
+            full_text = self.py3.composite_join(self.format_separator, data)
```

Comment:
just use what it is.

so here it is replacing the placeholder `{format_device}` so that is the __only__ thing that makes any sense 😄